tests/checkup_test.go

```diff
@@ -29,20 +29,39 @@ import (
 
 	batchv1 "k8s.io/api/batch/v1"
 	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
-	testServiceAccountName = "default"
-	testCheckupJobName     = "dpdk-checkup"
+	testServiceAccountName                     = "dpdk-checkup-sa"
+	testKiagnoseConfigMapAccessRoleName        = "kiagnose-configmap-access"
+	testKiagnoseConfigMapAccessRoleBindingName = testKiagnoseConfigMapAccessRoleName
+	testConfigMapName                          = "dpdk-checkup-config"
+	testCheckupJobName                         = "dpdk-checkup"
+
+	paramNUMASocket                      = "0"
+	paramNetworkAttachmentDefinitionName = "intel-dpdk-network-1"
```

Comment:
Please consider taking this parameter as an environment variable.

Reply:
DONE